roles/maintenance_14_sles/tasks/main.yml

```
@@ -0,0 +1,76 @@
+---
+
+- name: This task only serves as a template for the tasks below
+  ansible.builtin.set_fact:
+    ignoreme: &task
+      name: "{{ vars.taskid }}: {{ vars.name }}"
+      register: task
+      when:
+        - "vars.taskid not in maintenance_exclude_tasks"
+        - "maintenance_only is not defined or maintenance_only == vars.taskid"
+  vars:
+    taskid: ignoreme
+    name: bar
+
+- <<: *task
+  vars:
+    taskid: 14-001
+    name: "Apparmor: Check if apparmor is enabled and enforcing apparmor_status"
+  ansible.builtin.command: aa-status
+  register: apparmor_status_output
+  changed_when: false
+  failed_when: false
+
+- <<: *task
+  vars:
+    taskid: 14-001
+    name: "Apparmor: Check if apparmor is enabled and enforcing apparmor_status"
+  ansible.builtin.debug:
+    msg: "{{ apparmor_status_output.stdout }}"
+  when: apparmor_status_output is succeeded
+
+- <<: *task
+  vars:
+    taskid: 14-002
+    name: "Check if the system is properly registered at SUSE SUSEConnect --status-text"
+  ansible.builtin.command: SUSEConnect --status-text
+  register: suseconnect_status_output
+  changed_when: "'Not Registered' in suseconnect_status_output.stdout"
+
+- <<: *task
+  vars:
+    taskid: 14-003
+    name: "Zypper: Update package lists and check for errors zypper refresh"
+  community.general.zypper:
+    name: '*'
+    update_cache: true
+  register: zypper_refresh_output
+  failed_when: "{{ zypper_refresh_output.rc != 0 }}"
```

If you're not using the result of the task in another task, you don't actually need to register the task result again. All tasks using the `*task` "template" already register their result in the `task` variable, which can be used in e.g. `changed_when` or `failed_when`.

```suggestion
  failed_when: "task.rc != 0"
```

---

Wait, there actually is no `.rc` at all in the result of `community.general.zypper`. Have you tested this code?
I think `failed_when` can be omitted entirely in this case, just letting it fail when zypper says it failed.
```suggestion
```